### ISSUE TITLE
support geojson

### ISSUE THREAD

**Issue by robkooper** [plantbreeding/API#288]
It would be nice to use geojson instead of latitude and longitude. This would allow us to have a more complex way of describing the geospatial location.

Additionally to returning a latitude longitude, we can now also return:
```
{
  "type": "Feature",
  "geometry": {
    "type": "Point",
    "coordinates": [longitude, latitude]
  },
  "properties": {
    "name": "name"
  }
}
```

Benefit is that many mapping programs understand the geojson and can plot the information easily on a map.

For more information see http://geojson.org/

**Comment by dauglyon** [comment]
definite +1 for this, I suggest we keep the X/Y keys and info for row/col/other types of X/Y info and add a `geoJSONFeature` key or similar to obsUnits. It may also be good to have for locations

**Comment by cpommier** [comment]
Definitely an interesting propsal. We should double check the consequence on js librairies like leaflet. Even better, give hints on how to inject geojson in leaflet or other free mapping tools

**Comment by dlebauer** [comment]
“Leaflet is quite good at handling it [geojson]" https://leafletjs.com/examples/geojson/

**Comment by BrapiCoordinatorSelby** [comment]
Resolved in V2.0
GeoJSON now used for ObservationUnits, Locations, and Images

**Comment by dauglyon** [comment]
🎉
